tests/cpp/accuracy/CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ include(../cmake/common.cmake)
 
 find_package(OpenCV REQUIRED COMPONENTS core highgui videoio imgproc imgcodecs)
 
-set(ENABLE_PY_BINDINGS OFF)
+set(ENABLE_PY_BINDINGS ON)
```

Comment:
Can be off now, since we don't use the generated binaries, but instead build them with scikit-build